backend/internal/storage/postgres/works.go

```diff
@@ -35,13 +36,15 @@ const JoinedTable = `
 	users AS u ON wc.user_id = u.id
 	JOIN
 	assignment_outlines AS ao ON sw.assignment_outline_id = ao.id
+	JOIN
+	classrooms AS c ON ao.classroom_id = c.id
 `
 
 // squashes a list of student work contributors to a list of student works with an array of contributors
 func formatWorks[T models.IStudentWork, F models.IFormattedStudentWork](rawWorks []T, newFormattedWork func(T) F) []F {
```

Comment:
No change requested, making it known you cooked here

Reply:
why thank you kind sir